test/benchmarks/benchmarks.cpp

```diff
@@ -153,21 +176,23 @@ BENCHMARK_TEMPLATE(binop, uint256, uint256, sub);
 BENCHMARK_TEMPLATE(binop, uint256, uint256, inline_sub);
 BENCHMARK_TEMPLATE(binop, uint256, uint256, experimental::add_recursive);
 BENCHMARK_TEMPLATE(binop, uint256, uint256, experimental::add_waterflow);
-BENCHMARK_TEMPLATE(binop, uint256, uint256, mul);
-BENCHMARK_TEMPLATE(binop, uint256, uint256, mul_loop);
-BENCHMARK_TEMPLATE(binop, uint256, uint256, mul_loop_opt);
+BENCHMARK_TEMPLATE(binop, uint256, uint256, mul_);
+BENCHMARK_TEMPLATE(binop, uint256, uint256, mul_loop_);
+BENCHMARK_TEMPLATE(binop, uint256, uint256, mul_loop_opt_);
 BENCHMARK_TEMPLATE(binop, uint256, uint256, public_mul);
 BENCHMARK_TEMPLATE(binop, uint256, uint256, gmp::mul);
 
-BENCHMARK_TEMPLATE(binop, uint512, uint256, umul);
-BENCHMARK_TEMPLATE(binop, uint512, uint256, umul_loop);
+BENCHMARK_TEMPLATE(binop, uint512, uint256, umul_);
```

Comment:
Is
```
BENCHMARK_TEMPLATE(binop, uint512, uint512, umul_);
```
needed / makes sense?

Reply:
I don't think so it is useful. I like tuning this for EVM needs. The rest should just work. Even the 512x512 regular mul is a stretch. EVM top need is umul<256> for `MULMOD`.